spec/lib/approval_request_listener_spec.rb

```diff
@@ -0,0 +1,117 @@
+describe ApprovalRequestListener do
+  let(:client)               { double(:client) }
+  let(:event)                { ApprovalRequestListener::EVENT_REQUEST_FINISHED }
+  let(:approval_request_ref) { "0" }
+  let(:payload)              { {"request_id" => request_id, "decision" => decision, "reason" => reason } }
```

Comment:
Same here, `"request_id" => "1"`, since `request_id` isn't being used in any expectations, I don't see a benefit to having it split out into its own `let`.